values.yaml

```
@@ -673,7 +673,7 @@ server:
   # This can be used together with an OnDelete deployment strategy to help
   # identify which pods still need to be deleted during a deployment to pick up
   # any configuration changes.
```

Thanks for this contribution. I've just bumped into this as well.

Just a nitpick, because I find the comment misleading as it suggests that it's useful just in combination with OnDelete strategy. I actually find it more useful in combination with the other strategy.

```suggestion
  # This can be used together with an OnDelete deployment strategy to help
  # identify which pods still need to be deleted during a deployment or with
  # RollingUpdate on an auto-unsealed vault to pick up any configuration changes.
```
Since RollingUpdate and OnDelete are the only possible values of updateStrategy, the comment may very well be  removed as well.

---

@miminar Makes sense 👍

---

I think we mention OnDelete specifically since that's the recommended strategy to ensure standby's are updated before the active primary: https://developer.hashicorp.com/vault/tutorials/kubernetes/kubernetes-raft-deployment-guide#upgrading-vault-on-kubernetes

(See also the description on the original PR: https://github.com/hashicorp/vault-helm/pull/1001)